hw/ip/otbn/data/otbn.hjson

```diff
@@ -284,6 +284,33 @@
         }
       ]
     } // register : fatal_alert_cause
+    { name: "SEC_WIPE",
+      desc: '''
+        Trigger a secure wipe operation.
+
+        This register should only be accesed while OTBN is not busy, as indicated by the !!STATUS.busy flag.
+        Access attempts while OTBN is busy are ignored.
+      '''
+      swaccess: "wo",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0",
+          name: "dmem",
+          resval: 0,
+          desc: "Write 1 to securely wipe data memory. Always reads as 0."
```

Comment:
I don't think we document the expected read value for write-only registers. If you're force-pushing again, could you drop the second sentence for each of these fields?